charts/vsphere-cpi/v1.0.0/templates/vsphere-cloud-config-cm.yaml

```
@@ -6,11 +6,10 @@ metadata:
 data:
   vsphere.conf: |
     [Global]
-    secret-name = "vsphere-cpi-creds"
+    secret-name = {{ .Values.vCenter.credentialsSecretName | default "vsphere-cpi-creds" | quote }}
```

If I edit `vCenter.credentialsSecretName` and then decide to set `vCenter.generateCredentialsSecret=false`, we should use the generated secret's default name instead.

```suggestion
{{ if .Values.generateCredentialsSecret -}}
    secret-name = {{ .Values.vCenter.credentialsSecretName | quote }}
{{ else -}}
    secret-name = "vsphere-cpi-creds"
{{ end -}}
```